web-registry/src/components/molecules/MetadataJSONField.tsx

```diff
@@ -1,27 +1,23 @@
 import React, { useEffect, useState } from 'react';
-import { Field, useFormikContext } from 'formik';
+import { Field, useFormikContext } from 'formik'; // FormikContextType
```

Comment:
```suggestion
import { Field, useFormikContext } from 'formik';
```
Do you still want this comment?